kebechet/managers/thoth_advise/thoth_advise.py

```diff
@@ -380,16 +377,22 @@ def run(self, labels: list, analysis_id=None):
                     )
                     return False
                 _LOGGER.debug(json.dumps(res))
-
+                with open(".thoth.yaml", "r") as f:
+                    thoth_config = yaml.safe_load(f)
+                overlays_dir = thoth_config.get("overlays_dir")
                 self.runtime_environment = _runtime_env_name_from_advise_response(
                     res[0]
                 )
                 to_ret = False
                 if res[1] is False:
                     _LOGGER.info("Advise succeeded")
                     self._write_advise(res[0])
+                    res[0].update({"document_id": analysis_id})
+                    file_path = os.path.join(
+                        overlays_dir, str(self.runtime_environment), "Pipfile.lock"
```

Comment:
Hey, what happens if there is no overlays dir? `overlays_dir` will still be an empty string. This should looks something like:
```
file_path = os.path.join(overlays_dir, self.runtime_environment, "Pipfile.lock") if overlays_dir else "Pipfile.lock"
```

Reply:
Ack, thanks for pointing out